Automattic/jetpack
# Publicize: refresh connections once post saves

~### Branched off from https://github.com/Automattic/jetpack/pull/21107~

This PR refreshes the connections list once the post saves. For this purpose, it adds a new `usePostJustPublished()` hook.

#### Changes proposed in this Pull Request:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Create a new post.
* Open the Jetpack sidebar.
* Add social media connections.
* Enable some connections to share the post.
* Save the post.

Confirm the connections list gets updated properly.
